Feature/products

Romandionus opened the pull request:
добавил html разметку и часть сss

Romandionus committed:
feature products

add html and css without images

Romandionus committed:
Merge branch 'main' into feature/products

Romandionus committed:
feature/products

html + css

AlexandraKrosnik commented on src/sass/base/_common.scss:
```diff
@@ -23,17 +23,17 @@ a {
 }
 .container {
-  padding: 0 15px;
+  // padding: 0 15px;
```
Це ти видалив padding?

Romandionus replied:
это решил раскомментировав у себя

AlexandraKrosnik commented on src/sass/layout/_products.scss:
```diff
@@ -0,0 +1,94 @@
+.products__list {
+}
+.products__item {
+  margin-bottom: 130px;
```
А чому 130px, по макету наче 20px?

Romandionus replied:
обсуждали с тобой по макету таки 130

Romandionus committed:
feature/products v0.1

Romandionus committed:
feature/products v0.2

Romandionus committed:
feature/products v0.3

Romandionus commented:
залил комит с точками и картинкой поверх карточек

AlexandraKrosnik commented on src/sass/layout/_products.scss:
```diff
@@ -0,0 +1,187 @@
+    transform: translate(5%, -45%);
+    @media (min-device-pixel-ratio: 2), (min-resolution: 192dpi), (min-resolution: 2dppx) {
+      background-image: url(./../images/products/mobile-ice-cream@2x.png);
+    }
```
@media screen and (max-width: 290px) {
      transform: translate(0, -41%);
    }
Можна всім бефор дати таке медія правило і воно на менших розмірах ломатись не так сильно буде

Romandionus replied:
действительно помогло

AlexandraKrosnik commented on src/partials/products.html:
```diff
@@ -0,0 +1,44 @@
+        <ul class="products__list">
+            <li class="products__item">
+                <h3 class="products__name">ice cream</h3>
+                <span></span>
```
Краще спану дати клас, наприклад products__points чи products__dot і стилі писати для класу. Бо якщо десь на сайті буде спан ще один, щоб ці стилі до нього не підключились

Romandionus replied:
products__dot мне больше понравился

AlexandraKrosnik commented on src/sass/layout/_products.scss:
```diff
@@ -0,0 +1,187 @@
+  border-radius: 50%;
+  border: none;
+  padding: 10px;
+  cursor: pointer;
```
width: 40px;
  height: 40px;
  display: flex;
  align-items: center;
  justify-content: center;
  margin: 0 auto;
Краще замість падінгів такий варіант, Катерина через падінги вертала мені дз

Romandionus replied:
поправил

AlexandraKrosnik commented on src/sass/layout/_products.scss:
```diff
@@ -0,0 +1,187 @@
+  top: 45%;
+  left: 46%;
+  transform: translate(-200%, -200%);
+}
```
top: 42%;
  left: 42%;
 transform: translate(-50%, -50%);
  @media screen and (max-width: 228px) {
    top: 34%;
    left: 40%;    
  }

  @media screen and (min-width: 229px) and (max-width: 270px) {
    top: 39%;   
  }

Спробуй тут так, бо мені здається transform: translate(-200%, -200%); це виглядає дуже страшно

Romandionus replied:
исправил как ты рекомендовало стало лучше еще добавил display none до 235 пикселей

AlexandraKrosnik commented on src/sass/layout/_products.scss:
```diff
@@ -0,0 +1,187 @@
+  letter-spacing: 0.04em;
+  text-transform: uppercase;
+  margin-bottom: 144px;
+}
```
Спробуй додати тут якесь медіа правило, щоб на маленьких розмірах шрифт зменшувався, і текст не вилазив за межі. Денис нам наче показував як це можна зробити через відносні величини, або просто font-size: 19px; наприклад.

AlexandraKrosnik replied:
Там де заголовок milkshakes, воно теж трішки вилазить

Romandionus replied:
задал vw + flex карточкам

Romandionus committed:
feature/products v0.4

add media for max width 290 and lover
add class for span
add flex to button and for cards

Romandionus committed:
feature/products v0.4.1

add 2 class section__pre-title and section__title

Romandionus committed:
Merge branch 'main' into feature/products

Romandionus committed:
feature/products v0.4.2

comment my styles  product title and condition

AlexandraKrosnik committed:
Merge branch 'main' into feature/products
